sci-calculators/bc-gh/bc-gh-1.2.6.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://github.com/gavinhoward/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE=""
+
+S="${WORKDIR}/bc-${PV}"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+src_configure() {
+	local configure_call=(
+		./configure.sh
+		# This option is to turn off tests that require GNU bc
+		# to generate test cases.
+		-G
+	)
+	einfo "Running ${configure_call[@]}"
+	EXECSUFFIX="-gh" \
+	PREFIX="${EPREFIX}/usr" \
+	"${configure_call[@]}" || die
```

Comment:
Is all this complexity really necessary? Why wouldn't a single-liner 

`EXECSUFFIX="-gh" PREFIX="${EPREFIX}/usr" ./configure.sh -G || die` work here?

Of course you can export those variables into ENV before calling configure.sh if that's needed.

Reply:
I had checked a recently-update ebuild for how it did its configure step with a custom `configure.sh` script, and that is what it had. I thought that I would need to follow that. Sorry about that.

Changed to the one-liner.